src/did/Did.ts

```diff
@@ -80,7 +80,7 @@ export interface IDidService {
 
 export interface IDidDocument
   extends IDidDocumentCore,
-    Partial<IDidDocumentPpties> {}
+    Partial<IDidDocumentProperties> {}
```

Comment:
same

Reply:
just a consumer of IDidDocumentProperties